app/views/pages/home.html.erb

```diff
@@ -1,173 +1,182 @@
 <div id="top" class="home-banner">
```

Comment:
Featured trips were messy. The below cleans and re-organises. Also places each featured trip inside of an <a> tag such that clicking anywhere on the card opens the journey.